### DISCUSSION TITLE
Time resolution and frequency

### DISCUSSION THREAD

**Discussion by m-shak** [e2nIEE/pandapower#1081] [Q&A]
My name is Mohammad Shaklab and I am a research assistant at TU Berlin, Energy Systems Department.

Currently I am modelling a certain electric network using your tool, Pandapower, to see the real-time operation, system imbalances and frequency deviations. I was wondering if you can help me with some questions, where your answers would be very helpful to the team of energy systems department and to me.

1. Does Pandapower have the capability to generate results of the frequency? So far I have seen in most cases power is being analyzed but I have not see in any example the aimed output is the frequency.

2. Can I change the time resolution? To have a time series with second-based time steps? In most cases I have seen that the time series is used in a 15-min step or in hourly base.

I hope answering the two questions won't take so much time from you.

Best Regards

Mohammad

**Comment by lthurner** [comment]
1. No, because pandapower is made for static calculation, where frequency is always assumed to be constant.
2. The time series simulation is just a series of power flows, the resolution is up to your data. pandapower does not have any restriction on the time resolution (although at some point the results will not be realistic any more, you will need a RMS simulation, which pandapower does not provide).
